weave/trace_server/environment.py

```diff
@@ -38,6 +38,17 @@ def wf_clickhouse_max_memory_usage() -> Optional[int]:
         return None
 
 
+def wf_clickhouse_max_execution_time() -> Optional[int]:
+    """The maximum execution time for the clickhouse server."""
+    time = os.environ.get("WF_CLICKHOUSE_MAX_EXECUTION_TIME")
+    if time is None:
+        return None
+    try:
+        return int(time)
+    except ValueError:
+        return None
```

Comment:
Do we want to silence incorrect values?

Reply:
I think silently failing here is better than crashing the pod if the var is incorrect. It's better to not apply the fix than brick I think here...